src/components/Header.js

```diff
@@ -1,9 +1,27 @@
 import React from "react";
+import { Link } from "react-router-dom";
 
 export default function Header() {
   return (
     <header className="ui centered">
       <h1 className="ui center">Rick &amp; Morty Fan Page</h1>
+      <div>
+        <Link to={`/`}>
+          <a href="">Home</a>
```

Comment:
`<Link>` under the hood is an `<a>` tag. So you can do this instead => ` <Link to='/'>Home</Link>`